app/db/index.js

```diff
@@ -4,20 +4,26 @@ const { readFileSync } = require('fs');
 const { database, server } = require('../config');
 
 const connect = () => {
-  const { name, username, password, host, dialect, port } = database;
+  const { URL, name, username, password, host, dialect, port } = database;
   const config = {
-    host,
-    dialect,
-    port,
-    logging: false,
+    connection: () => {
+      if (URL) return URL;
+      return `postgres://${username}:${password}@localhost:5432/${name}`;
```

Comment:
`localhost:5432` hardcoded? Shouldn't this come from the env config? Or am I misreading this.

Reply:
I didn't even see this one :smile: